### PR TITLE
feat(docker): Add healthcheck

### PR DESCRIPTION
**Pull request by ThomasSanson**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,9 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/* && \
     apt-get clean
 
+HEALTHCHECK --interval=1m --timeout=30s --retries=3 \
+  CMD curl --fail http://localhost/8080 || exit 1
```
Comment (tchiotludo) [inline Dockerfile:11]:
```suggestion
  CMD curl --fail http://localhost:8080/health || exit 1
```

**Review by tchiotludo — CHANGES_REQUESTED** [review]
Seems that there is a mistake on the url

File: Dockerfile (hunk 1 above)
Comment (ThomasSanson) [reply Dockerfile]: indeed 😅

**Comment by sandropace** [comment]
healthchecks are behind authentication.  that should be outside of auth, so that k8s can still verify health of the container running akhq.

**Comment by tchiotludo** [comment]
No, healthcheck are on separated port with no auth

**Pull request merged**